modules/shorewall/manifests/four/rule.pp

```diff
@@ -67,4 +67,12 @@
     ],
   }
 
+  shorewall::rule {
+'outgoing-mail':
+    source => 'all',
+    destination => 'net',
+    action  => 'SMTP(DROP)',
```

Comment:
Das blockiert auch das SMTP, das vom Server selbst initiiert ist.
Außerdem ist das in der falschen Datei. Das sollte iwo nach supornode.pp